src/rpc/rpcevo.cpp

```diff
@@ -792,15 +811,29 @@ UniValue bls_generate(const JSONRPCRequest& request)
     return ret;
 }
 
-UniValue _bls(const JSONRPCRequest& request)
+void _bls_help()
 {
-    if (request.params.empty()) {
-        throw std::runtime_error(
-                "bls \"command\" ...\n"
-        );
-    }
+    throw std::runtime_error(
+            "bls \"command\" ...\n"
+            "Set of commands to execute BLS related actions.\n"
+            "To get help on individual commands, use \"help bls command\".\n"
+            "\nArguments:\n"
+            "1. \"command\"        (string, required) The command to execute\n"
+            "\nAvailable commands:\n"
+            "  generate          - Create a BLS secret/public key pair\n"
+            );
+}
 
-    std::string command = request.params[0].get_str();
+UniValue _bls(const JSONRPCRequest& request)
```

Comment:
Same here, let's use `bls()` instead.

Reply:
The reason I used `_bls()` initially was that `bls()` conflicts with the external namespace `bls`. I then decided to use an underscore for `_bls_help` to maintain consistency.

Reply:
Yes, this was one of the naming things I was not sure about. I did not want to rename this one and wanted to maintain consistency (thus the `_bls_help()`). What do you guys prefer?

Reply:
Leading underscore is reserved for global implementation values in C++, it doesn't really matter in our case, but may be it is better to use `bls_` instead, for example?

Reply:
Haha, how ironic -- naming conflicts due to namespaces. 🙈 Could use trailing I guess, my preference would be `bls_cmd()` or the like.

Regardless, I think we should put a comment directly above explaining that the bls namespace causes conflicts with the name `bls()` and this is why the discrepancy for this particular method name. And I think `bls_help()` should still align (no leading/trailing underscores).

Reply:
I renamed to `bls_help()`. I don't think this PR should mess with `_bls()` since that would affect additional things unrelated to adding detailed help.

Reply:
Btw, are there going to be a lot of subcommands for `bls`? Can't we just go with `blsgenerate()`, `blssmthelse()` etc.?

Reply:
> Can't we just go with `blsgenerate()`, `blssmthelse()` etc.?

Not sure on bls commands, but re: syntax, what you suggested seems less clean to me (brain has to pick apart the words), and doesn't really keep w/convention in gobject (have not checked masternode yet). Personally I find the underscores (or camelCase) easier to parse visually.

Reply:
@thephez I agree that renaming `_bls()` mixes adding a feature w/a simple refactoring/cleanup, technically.

Reply:
@nmarley ok, smth like `getnewblspair()` similar to `getnewaddress()` etc.? The point is - if there won't be many `bls()` subcommands (or if they are going to have completely different use cases) then we might just use longer names instead. But that's out of the scope for this specific PR ofc, just a random thought :)